forge-gui/res/cardsfolder/upcoming/cait_cage_brawler.txt

```diff
@@ -0,0 +1,14 @@
+Name:Cait, Cage Brawler
+ManaCost:R G
+Types:Legendary Creature Human Warrior
+PT:1/1
+S:Mode$ Continuous | Affected$ Card.Self | AddKeyword$ Indestructible | Condition$ PlayerTurn | Description$ As long as it's your turn, CARDNAME has indestructible.
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ Whenever NICKNAME attacks, you and defending player each draw a card, then discard a card. Put two +1/+1 counters on NICKNAME if you discarded the card with the highest mana value among those cards or tied for highest.
+SVar:TrigDraw:DB$ Draw | Defined$ You & TriggeredDefender | NumCards$ 1 | SubAbility$ DBDiscard
+SVar:DBDiscard:DB$ Discard | Mode$ TgtChoose | Defined$ You & TriggeredDefender | RememberDiscarded$ True | SubAbility$ FindHighestCMC
+SVar:FindHighestCMC:DB$ Pump | ImprintCards$ ValidAll Card.greatestRememberedCMC | SubAbility$ DBPutCounter
```

Comment:
This line might be unnecesairy ?

Reply:
yes

Reply:
Hmm any thoughts on the variable for the condition ?

Reply:
you should check `ConditionDefined$ Remembered` for the Valid

Reply:
Should the conditionpresent$ contain GreatestCMC or greatestRememberedCMC ?

Reply:
you'll want the CardProperty you've been using